lib/data_migrate/config.rb

```diff
@@ -12,12 +12,13 @@ def config
   end
 
   class Config
-    attr_accessor :data_migrations_path, :data_template_path, :db_configuration, :spec_name
+    attr_accessor :data_migrations_path, :data_migrations_gen_path, :data_template_path, :db_configuration, :spec_name
 
     DEFAULT_DATA_TEMPLATE_PATH = "data_migration.rb"
 
     def initialize
       @data_migrations_path = "db/data/"
+      @data_migrations_gen_path = "db/data"
```

Comment:
what does `gen` stand for? do you mind adding it to the docs as well. also, please rebase with `main`, at some point travisci tests stopped running to i switched to github actions